### DISCUSSION TITLE
scMACS::getDifferentialAccessibleTiles errors

### DISCUSSION THREAD

**Discussion by ziyuanhe** [aifimmunology/MOCHA#51] [Announcements]
I was able to run differential peaks with no issues but the same code returned the error below. i think it happened since i update Mocha last week or the week before. 
![Screen Shot 2022-09-20 at 5 02 56 PM](https://user-images.githubusercontent.com/79122830/191385834-1f8420dc-10b4-4d35-88e7-720822bf7f06.png)
I double-checked that NAtoZero = TRUE was set for  scMACS::getSampleTileMatrix and also when i check `cd4na_Matrix <- scMACS::getCellPopMatrix(SampleTileMatrices, "cd4_naive")
cd4na_Matrix[is.na(cd4na_Matrix)]` there were no NA in the matrix.

**Comment by markphillippebworth** [comment]
@imran-aifi : This bug is fixed already within the BugFix branch. 

@ziyuanhe : Try installing from the BugFix branch.

**Comment by imran-aifi** [comment]
@markphillippebworth fixed in 71dc97a9f2918ebd6a77e68dfd6777d48144542b
